massa-consensus-worker/src/worker/init.rs

```diff
@@ -225,6 +232,13 @@ impl ConsensusWorker {
                     genesis_block_ids.iter().map(|h| (*h, 0)).collect();
                 write_shared_state.best_parents =
                     genesis_block_ids.iter().map(|v| (*v, 0)).collect();
+                for (thread, b) in genesis_block_ids.iter().enumerate() {
```

Comment:
no need to bother with this, we discard incoming blocks that are at or before the genesis period

Reply:
Done.